src/main/java/com/elastic/support/util/ArchiveUtils.java

```diff
@@ -110,6 +109,13 @@ public void extractDiagnosticArchive(String sourceInput, String targetDir, boole
             String name = tae.getName();
             name = name.replace(baseArchivePath, "");
 
+            if(  name.contains("gc.log.") ||
+                    (name.contains(".log") && !name.equalsIgnoreCase("diagnostics.log") ) ||
```

Comment:
Will this still work on 7.x JSON-based logs?

Reply:
No, I haven't started on that yet. They won't be collected as of now. When I start on that I will need to address that here.